kornia/color/rgb.py

```diff
@@ -1,5 +1,53 @@
 import torch
 import torch.nn as nn
+from typing import Union
+
+
+class RgbToRgba(nn.Module):
+    r"""Convert image from RGB to RGBA.
+
+    Add an Alpha channel to existing RGB image.
+
+    args:
+        image (torch.Tensor): RGB image to be converted to RGBA.
+        alpha_val (int): An integer between 0 to 255.
+
+    returns:
+        torch.Tensor: RGBA version of the image.
+
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 4, H, W)`
+
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgba = kornia.color.RgbToRgba(alpha_val)
+        >>> output = rgba(input)  # 2x4x4x5
+    """
+
+    def __init__(self, alpha_val: Union[float, torch.Tensor]) -> None:
+        super(RgbToRgba, self).__init__()
+        self.alpha_val: Union[float, torch.Tensor] = alpha_val
+
+    def forward(self, image: torch.Tensor, alpha_val: Union[float, torch.Tensor]) -> torch.Tensor:  # type: ignore
+        return rgb_to_rgba(image, self.alpha_val)
+
+
+def rgb_to_rgba(image: torch.Tensor, alpha_val: Union[float, torch.Tensor]) -> torch.Tensor:
+    if not torch.is_tensor(image):
+        raise TypeError("Input type is not a torch.Tensor. Got {}".format(type(image)))
+    if len(image.shape) < 3 or image.shape[-3] != 3:
+        raise ValueError("Input size must have a shape of (*, 3, H, W).Got {}"
+                         .format(image.shape))
+    # add one channel
+    r, g, b = torch.chunk(image, image.shape[-3], dim=-3)
+    if isinstance(alpha_val, float):
+        a: torch.Tensor = torch.full_like(r, fill_value=float(alpha_val))
+    else:
+        a: torch.Tensor = alpha_val
```

Comment:
I was assuming one alpha value per batch. Maybe it has to be clear in the documentation. For me it makes sense to have three options:

1. Single float value: all the images have the same alpha value, by default 0.
2. Torch tensor of size batch. It means you specify a value that is used to fill the alpha channel for each image.
3. Torch tensor of size BxWxH: It means you provide the whole channel per each image (maybe also W x H make sense, it provides the same alpha value to all the images allowing to change it spatial-wise)